ouroboros-network/src/Ouroboros/Network/Block.hs

```diff
@@ -139,12 +157,14 @@ blockPoint b =
 --
 data ChainUpdate block = AddBlock block
                        | RollBack (Point block)
-  deriving (Eq, Show)
+
```

Comment:
As an aside, the consensus package has a chain update type like:
```
data ChainUpdate blk a =
    AddBlock a
  | RollBack (Point blk)
  deriving (Show, Functor)
```
This allows the point type and block type to be independent. This is useful when you have headers or blocks, but they both share the same point type.

We might want to move that version into the network package.